### PR TITLE
resolved conflict

### PR DESCRIPTION
**Pull request by Adaking-slack**

### CODE REVIEW

File: names.txt [hunk 1]
```diff
@@ -3,6 +3,13 @@ Abdulbasit Yusuf
 Allename Anthony
 Emmanuel Oloke
 Mamodu Joshua
+<<<<<<< HEAD
```
Comment (oseunabiola) [inline names.txt:6]: Remove this line and other ones with non-name characters.

File: names.txt [hunk 2]
```diff
@@ -13,6 +17,10 @@ Ejiroghene Laurel Dafe
 Adeyomola Kazeem
 Kelechi Joanna Ojelabi
 Austin Wopara
+Adetayo Akinsanya Samuel
+Austin Wopara
```
Comment (oseunabiola) [inline names.txt:22]: There are duplicated names.

File: names.txt (hunk 1 above)
Comment (Adaking-slack) [reply names.txt]: done

File: names.txt (hunk 2 above)
Comment (Adaking-slack) [reply names.txt]: done

**Comment by Oluwasetemi** [comment]
Please I am sorry for all the trouble we have caused you but count it all joy. Your PR is about to be merged and you will be able to see your name on this [url](https://altschool-opensource-names.oluwasetemi.dev/). 

To enable me to do this, **READ the README again carefully**, update your code with the latest and push the update. I am here for you for the next **6 hours** before I close the PR so you can initiate a fresh PR which will be easier to merge.

**Comment by Succynice** [comment]
Sorry I have to close this PR, kindly update your code and initiate a fresh PR

Cheers 🥂

**Pull request closed without merging**